site/_docs_1_0/httpproxy.md

```diff
@@ -990,6 +990,31 @@ spec:
       weight: 20
 ```
 
+## Upstream Validation
+
+When defining upstream services on a route, it's possible to configure the connection from Envoy to the backend endpoint to communicate over TLS.
```

Comment:
If both the `caSecret` and `subjectName` keys are required, we should state that explicitly.